.travis.yml

```diff
@@ -14,7 +14,12 @@ before_install:
     #- cat install.txt | xargs sudo apt-get install -y
   - sudo apt-get install -y clang-3.9 clang-format-3.9 clang-tidy-3.9 flawfinder libxml2 pylint python-git python-yapsy findbugs
   - sudo -H pip install -r requirements.txt
+  - sudo pip install sphinx travis-sphinx
```

Comment:
These should be added to `requirements.txt`.

Reply:
Actually, I can see these being separate since they are only used by Travis, not by typical developers.